pom.xml

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jboss</groupId>
+    <artifactId>jboss-parent</artifactId>
+    <version>30</version>
+  </parent>
+
+  <groupId>org.kie</groupId>
+  <artifactId>kie-gwthelper-maven-plugin</artifactId>
+  <packaging>maven-plugin</packaging>
+  <version>${version.org.kie.gwthelper.maven}</version>
```

Comment:
Is this still needed? I mean to have this version in a property.

Reply:
it should not be needed for a simple project